homeassistant/components/roborock/strings.json

```diff
@@ -454,6 +454,18 @@
     "map_failure": {
       "message": "Something went wrong creating the map"
     },
+    "map_data_update": {
```

Comment:
I don't think that every single case should necessarily have a separate error message or translation. One way to think about this can be: Will the user do something different with additional level of detail? I think it would be good to have a more generic error message then  use traditional logging to put more detail if needed.